IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/wolfssl_tsip_unit_test.c

```diff
@@ -0,0 +1,1020 @@
+    if (xRet == pdPASS)
+        xRet = xTaskCreate(tskSha256_Test, "sha256_test", 
+                            SMALL_STACK_SIZE, NULL, 3, NULL);
+#endif
+
+    if (xRet == pdPASS) {
+        printf(" Waiting for completing tasks ...");
+        vTaskDelay(10000/portTICK_PERIOD_MS);
+        exit_loop = true;
+
+        for (i = 0; i < num; i++) {
+            if(!xSemaphoreTake(exit_semaph, 2000/portTICK_PERIOD_MS)) {
+                printf("exit semaphore not released by test task");
+                ret = -1;
+                xRet = pdFAIL;
+            }
+        }
+    }
+    
+    vSemaphoreDelete(exit_semaph);
+    
+    if ((xRet == pdPASS) &&
+       (sha_multTst_rslt == 0 && sha256_multTst_rslt == 0))
+        ret = 0;
+    else
+        ret = -1;
+    
+    RESULT_STR(ret)
+
+    return ret;
+}
+
+
+int tsip_crypt_AesCbc_multitest()
+{
+    int ret = 0;
+    int num = 0;
+    int i;
+    Info info_aes1;
+    Info info_aes2;
+    Info info_aes256_1;
+    Info info_aes256_2;
+    BaseType_t xRet;
+    
+#if defined(HAVE_AES_CBC) && defined(WOLFSSL_AES_128)
+    num+=2;
+#endif
+#if defined(HAVE_AES_CBC) && defined(WOLFSSL_AES_256)
+    num+=2;
+#endif
+    exit_loop = 0;
+    Aes128_Cbc_multTst_rslt = 0;
+    Aes256_Cbc_multTst_rslt = 0;
+
+    exit_semaph = xSemaphoreCreateCounting(num, 0);
+    xRet = pdPASS;
+    
+#if defined(HAVE_AES_CBC) && defined(WOLFSSL_AES_128)
+    XMEMCPY(&info_aes1.aes_key, &g_user_aes128_key_index1,
+                                            sizeof(tsip_aes_key_index_t));
+    xRet = xTaskCreate(tskAes128_Cbc_Test, "aes_cbc_tes1t",
+            SMALL_STACK_SIZE, &info_aes1, 3, NULL);
+#endif
+#if defined(HAVE_AES_CBC) && defined(WOLFSSL_AES_128)
+    XMEMCPY(&info_aes2.aes_key, &g_user_aes128_key_index2,
+                                            sizeof(tsip_aes_key_index_t));
+    if (xRet == pdPASS)
+        xRet = xTaskCreate(tskAes128_Cbc_Test, "aes_cbc_test2",
+                SMALL_STACK_SIZE, &info_aes2, 3, NULL);
+#endif
+
+#if defined(HAVE_AES_CBC) && defined(WOLFSSL_AES_256)
+    XMEMCPY(&info_aes256_1.aes_key, &g_user_aes256_key_index1,
+                                            sizeof(tsip_aes_key_index_t));
+    if (xRet == pdPASS)
+        xRet = xTaskCreate(tskAes256_Cbc_Test, "aes256_cbc_test1",
+                SMALL_STACK_SIZE, &info_aes256_1, 3, NULL);
+#endif
+#if defined(HAVE_AES_CBC) && defined(WOLFSSL_AES_256)
+    XMEMCPY(&info_aes256_2.aes_key, &g_user_aes256_key_index2,
+                                            sizeof(tsip_aes_key_index_t));
+    if (xRet == pdPASS)
+        xRet = xTaskCreate(tskAes256_Cbc_Test, "aes256_cbc_test2",
+                SMALL_STACK_SIZE, &info_aes256_2, 3, NULL);
+#endif
+    
+    if (xRet == pdPASS) {
+    printf(" Waiting for completing tasks ...");
+        vTaskDelay(10000/portTICK_PERIOD_MS);
+        exit_loop = true;
+
+        for (i = 0; i < num; i++) {
+            if(!xSemaphoreTake(exit_semaph, 2000/portTICK_PERIOD_MS)) {
```

Comment:
Please add single space after `if` here and for similar lines below.

Reply:
Fixed